### PR TITLE
Insurance info block

### PR DESCRIPTION
**Pull request by palmenhq**
Screenshot 2020-05-26 at 12 21 13
Screenshot 2020-05-26 at 12 35 25

### CODE REVIEW

File: src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            href={values.presale_info_link}
+            target="_blank"
+            rel="noopener nofollow"
+          >
+            {values.presale_info_link_text}↗
+          </Link>
+        </Column>
+
+        <Column halfHardBottom>
+          <Cta href={values.cta_link}>{values.cta_text}</Cta>
```
Comment (gustaveen) [inline src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx:172]: I think we can use the Button component here :) 
```suggestion
          <ButtonLinkBrandPivot href={values.cta_link} fullWidth styleType="outlined">
            {values.cta_text}
          </ButtonLinkBrandPivot>
```

**Comment by gustaveen** [comment]
Really nice! 🎉 I've added purple300 to the brand repo and added it here 22a86654903fd90ae3b01041f24c37d37f261694 so if you could update the color picker in storyblok with that option that would awesome! 😍

File: src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    padding: 7rem 0;
+  }
+`
+
+const StyledContentWrapper = styled(ContentWrapper)`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  & {
+    max-width: calc(784px + 2 * 2rem);
```
Comment (gustaveen) [inline src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx:28]:
```suggestion
    max-width: calc(49rem + 2 * 2rem);
```

File: src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  &:focus {
+    color: inherit;
+    text-decoration: underline;
+  }
+`
+
+const Cta = styled.a`
+  display: inline-block;
+  border: 1px solid ${colorsV3.gray900};
+  border-radius: 8px;
```
Comment (gustaveen) [inline src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx:91]:
```suggestion
  border-radius: 0.5rem;
```

**Review by gustaveen — DISMISSED** [review]
Feel free to update the cta if you want to :)

File: src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx (hunk 1 above)
Comment (palmenhq) [reply src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx:172]: oh sorry missed this comment 😅

**Review by gustaveen — APPROVED** [review]
🎉🎉🎉

**Pull request merged**